traffic_ops/traffic_ops_golang/auth/authorize.go

```diff
@@ -42,6 +42,26 @@ type CurrentUser struct {
 	TenantID     int            `json:"tenantId" db:"tenant_id"`
 	Role         int            `json:"role" db:"role"`
 	Capabilities pq.StringArray `json:"capabilities" db:"capabilities"`
+	perms        map[string]struct{}
```

Comment:
Any reason `perms` is lowercase?

Reply:
yes, so that it isn't exported. That way no code outside of the auth package can mutate it, so nothing can accidentally give a user more Permissions than they actually have.